docs/source/getting_started/projects.rst

```diff
@@ -2,6 +2,11 @@
 Projects
 ========
 
+.. warning::
+    Once your deployment is updated to the Teams version of the Citrine Platform, project behavior will change. Details can be found in the :doc:`Teams FAQ <../FAQ/team_management_migration>`.
```

Comment:
Per previous discussion of (unofficial) 80, 100, 120-character line length, Can we break this into two shorter lines?